templates/admin/judge/problem/change_form.html

```diff
@@ -2,19 +2,25 @@
 {% load i18n %}
 
 {% block extrahead %}{{ block.super }}
-    <script>
+    <script type="text/javascript">
         django.jQuery(function ($) {
             $('.submissions-link').appendTo('div#bottombar').show();
+            $('.problem-votes-link').appendTo('div#bottombar').show();
         });
     </script>
 {% endblock extrahead %}
 
 {% block after_field_sets %}{{ block.super }}
     {% if original %}
-        <a style="display: none" title="{% trans "View Submissions" %}" class="button submissions-link"
+        <a style="display: none" title="{{ _('View submissions') }}" class="button submissions-link"
            href="{% url 'admin:judge_submission_changelist' %}?problem__code={{ original.code }}">
             <i class="fa fa-lg fa-search-plus"></i>
-            <span class="text">{% trans "View submissions" %}</span>
+            <span class="text">{{ _('View submissions') }}</span>
```

Comment:
Why this change? I can't even remember if this is a Django template or Jinja2.

Reply:
Consistency with the rest of `/templates`. The only uses of `{% trans "Foo" %}` are in `/templates/admin`.

Reply:
Yeah that would be because `templates/admin` are rendered as Django and not Jinja2. I am surprised this even works.